test/benchmark/helpers.ts

```diff
@@ -13,7 +13,7 @@ governing permissions and limitations under the License.
 import { html, LitElement, render, TemplateResult } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 import '@spectrum-web-components/theme/sp-theme.js';
-import '@spectrum-web-components/theme/scale-medium.js';
+import '@spectrum-web-components/theme/scale-large.js';
```

Comment:
Just curious - why switch to large scale for benchmarking?

Reply:
It's not so much a switch as a correction. https://github.com/adobe/spectrum-web-components/pull/1971/files#diff-737633c0f2c82bcb5bc7453285cb9547b8d34ac0a4aae636c7ca02410ebb12d2R126 I could fix this separate from this work, but in debugging I found that they didn't match and wanted them to in order to better understand the test results.

Reply:
Gotcha! 👍